addons/zeus/functions/fnc_moduleSimulation.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: Fisher
+ * Toggle Simulation on object.
+ *
+ * Arguments:
+ * 0: The module logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+ 
+ #include "script_component.hpp"
+ 
+ params ["_logic"];
+ 
+ if !(local _logic) exitWith {};
```

Comment:
Module will be local to zeus client. However in this case of this module, we may want to actually exploit the module framework and make it only run on the server (then use the simulationGlobal command).

To do so, in the module's cfgVehicles entry you simple set `isGlobal = 0;`